validators/changing-node-ip.adoc

```diff
@@ -1,31 +1,21 @@
 = Changing a Node IP
 
-You can't change your node IP address, but what's possible is to remove the node with the incorrect/old IP address and register a new node under the correct/new IP while keeping the old SGX key. Please contact the core team before proceeding for further guidance.
+It's possible to change node IP address. But you it requires SKALE manager transaction that can be performed only by SKALE foundation. So you need to ask core team.
```

Comment:
It's possible to change the node IP address. But it requires a SKALE manager transaction that can be performed only by the SKALE foundation. So you are required to ask the core team.

Reply:
addressed